tests/run/cpp_stl_string_view.pyx

```diff
@@ -35,6 +35,25 @@ def test_return_to_py():
     return return_to_py()
 
 
+def test_coercion_py(py_str):
+    """
+    >>> b_asdf_5 = b_asdf * 5
+    >>> test_coercion_py(b_asdf_5)
+    """
+    if not isinstance(py_str, bytes):
+        return
+
+    cdef string_view sv1 = py_str
```

Comment:
Looks like this doesn't work out of the box. FWIW `std::string` does

Maybe we can try typing it as `bytes`

```suggestion
    cdef string_view sv1 = <bytes>py_str
```